docs/about/contributing.md

```diff
@@ -4,24 +4,24 @@ Thank you for considering contributing! There are many ways you can help.
 
 ## Issues
 
-File an issue if you think you've found a bug. Be sure to describe
+File an issue if you think you've found a bug. Be sure to describe:
 
-1. How can it be reproduced?
-2. What did you expect?
-3. What actually occurred?
-4. Version, platform, etc. if possibly relevant.
+1. How it can be reproduced
+2. What did you expect to happen?
+3. What actually happened?
+4. Version, platform, etc. if relevant
 
-You can file all issues with Screwdriver in the [screwdriver repo][api-issues-url]. We will update any issues we're working on with a daily summary. To see what we're currently working on, you can check out our [digital scrum board](https://github.com/screwdriver-cd/screwdriver/projects/4) in the Projects section in the [Screwdriver API repo][api-repo].
+You can file all issues with Screwdriver in the [screwdriver repo][api-issues-url]. We will update any issues we're working on with a daily summary. To see what we're currently working on, you can check out our [digital scrum board](https://github.com/screwdriver-cd/screwdriver/projects/4) in the Projects section of the [Screwdriver API repo][api-repo].
```

Comment:
Can we mentioned something about the label for the issue? That will make it easier for us to review the issues